### ISSUE TITLE
browser: [windows] IE11 got error: Object doesn't support property or method 'assign' for browser client

### ISSUE THREAD

**Issue by wey-gu** [minio/minio#3898]
## Expected Behavior
minio-server web-browser client got error from IE11 , which blocked the loading of initial authentication page.

## Current Behavior
Below is errors from IE11 F12 mode:
```
SCRIPT438: Object doesn't support property or method 'assign'
File: index_bundle-2017-02-15T22-41-23Z.js, Line: 30, Column: 27472
```

## Possible Solution

**simular fault:** https://github.com/mozilla-services/react-jsonschema-form/issues/206

## Steps to Reproduce (for bugs)

1. Deploy minio.exe for windows 64 bit
2. Access <serverIP>:9000 via IE 11

## Your Environment

* Version used:
```
Version: 2017-02-16T01:47:30Z
Release-Tag: RELEASE.2017-02-16T01-47-30Z
Commit-ID: 3d98311d9f4ceb78dba996dcdc0751253241e697
```
* Environment name and version : Only minio.exe without http proxy.
* Server type and version: refer to version above
* Operating System and version: Windows 7 64bit
* Link to your project: n/a

**Comment by krishnasrinivas** [comment]
https://github.com/minio/minio/issues/3791

**Comment by harshavardhana** [comment]
Can you validate the fix in #3884 on IE11 @rushenn ?

**Comment by wey-gu** [comment]
@harshavardhana sorry for bothering, I downloaded the binary minio server for [windows x64](https://dl.minio.io/server/minio/release/windows-amd64/minio.exe) while it's still old version, When could that download link be updated please?

**Comment by harshavardhana** [comment]
> @harshavardhana sorry for bothering, I downloaded the binary minio server for windows x64 while it's still old version, When could that download link be updated please?

We will update it soon @littlewey ..

**Comment by wey-gu** [comment]
@krishnasrinivas sorry it seems the mino.exe built on 2017-3-16 was still with this issue, it's still blocked in login interface. Should I raise another issue?

```
SCRIPT438: Object doesn't support property or method 'assign'
File: index_bundle-2017-03-15T17-49-51Z.js, Line: 30, Column: 22288
```

```
Version: 2017-03-16T21:50:32Z
Release-Tag: RELEASE.2017-03-16T21-50-32Z
Commit-ID: 5311eb22fd681a8cd4a46e2a872d46c2352c64e8
```

**Comment by krishnasrinivas** [comment]
@rushenn  can you try adding
```
{
  "plugins": ["transform-object-assign"]
}
``` 
to .babelrc? ping me for access to the windows machine. I can see the bug on IE11

reference: https://babeljs.io/docs/plugins/transform-object-assign/

**Comment by rushenn** [comment]
@krishnasrinivas checking

**Comment by krishnasrinivas** [comment]
@rushenn :
```
diff --git a/browser/app/index.js b/browser/app/index.js
index 26a1cc1..3afa865 100644
--- a/browser/app/index.js
+++ b/browser/app/index.js
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+import 'babel-polyfill'
+
 import './less/main.less'
 
 import React from 'react'
```

this was the only way I could make it work. The webpack config solution was not working on IE11 for some reason.

You can undo `transform-object-assign` related changes.

**Comment by rushenn** [comment]
@krishnasrinivas https://github.com/minio/minio/pull/3942

**Comment by wey-gu** [comment]
Dear all,
Any update on this please?
Thanks!
BR//Wey

**Comment by harshavardhana** [comment]
Master branch should be already fixed.

**Comment by wey-gu** [comment]
Dear @harshavardhana sorry for bothering when could I expect the binary version with the issue fixed? 

Many thanks!

**Comment by harshavardhana** [comment]
> Dear @harshavardhana sorry for bothering when could I expect the binary version with the issue fixed?

@littlewey it should be soon, we have been waiting on testing.  You could use docker btw 

```
docker pull minio/minio:edge
```

To get the latest UI changes until we make a new release.

**Comment by wey-gu** [comment]
@harshavardhana thanks a lot, the docker for windows requires win10 Pro(with hyper-v). I have to wait for the binary windows release. Looking forward to seeing its update .

BR//Wey

**Comment by wey-gu** [comment]
Update: best way to install docker in windows 7
choco install docker with [chocolatey](https://chocolatey.org/)

**Comment by harshavardhana** [comment]
> Update: best way to install docker in windows 7
> choco install docker with chocolatey

We have made the release.. btw you don't need to use `minio/minio:edge`  anymore.

**Comment by wey-gu** [comment]
@harshavardhana  Thanks, updated to latest released binary one, IE confirmed works perfectly fine.
